proto/redpanda/api/console/v1alpha1/debug_bundle.proto

```diff
@@ -0,0 +1,155 @@
+syntax = "proto3";
+
+package redpanda.api.console.v1alpha1;
+
+import "buf/validate/validate.proto";
+import "google/protobuf/timestamp.proto";
+
+// Request to start the creation of debug bundle process with given configuration parameters.
+message CreateDebugBundleRequest {
+  // Optional broker IDs. Do not set / leave empty to create for all.
+  repeated int32 broker_ids = 1;
+
+  // The size limit of the controller logs that can be stored in the bundle.
+  int32 controller_logs_size_limit_bytes = 2 [(buf.validate.field).int32 = {gte: 0}];
+
+  // For how long to collect samples for the CPU profiler
+  int32 cpu_profiler_wait_seconds = 3 [(buf.validate.field).int32 = {
+    gte: 15
+    lte: 600
+  }];
```

Comment:
quesiton: where did these limits come from?

Reply:
The minimum comes from from `rpk debug bundle --help`

```sh
--cpu-profiler-wait duration          For how long to collect samples for the CPU profiler (e.g. 30s, 1.5m). Must be higher than 15s (default 30s)
```

Regarding the max... I am not sure I thought I read somewhere or it was said somewhere that max would be `10m` or maybe that just seemed like a reasonable max value. But I can't find it now... So I have removed the `lte` validation.

Reply:
👍 thanks!